lib/gracenote/response/genre.rb

```diff
@@ -0,0 +1,13 @@
+require "gracenote/response/base"
+
+module Gracenote
+  class Response
+    class Genre < Base
+      self.attributes = %w[id num]
```

Comment:
__[rubocop]__
* `%w`-literals should be delimited by `(` and `)`. [:ref](https://github.com/bbatsov/ruby-style-guide#percent-literal-braces)